client/src/pages/Dashboard/MediaInfo/MediaItemDetails.tsx

```diff
@@ -0,0 +1,20 @@
+function MediaItemDetails() {
+    return (
+        <div className="flex grow w-full px-7">
+            <div className="flex h-full justify-center w-1/12">
+                <div className="bg-black w-7 h-7 mt-11"></div>
```

Comment:
we can also do the shorthand on this scenario (i.e `div className="bg-black w-7 h-7 mt-11" />`) if we don't have anything to put inside